roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,20 +15,27 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
     docker_image_base = {
         "origin": {
             "repo": "openshift",
             "image": "origin",
+            "has_registry_console": False,
```

Comment:
@rhcarvalho 
> Both methods containerized_docker_images and rpm_docker_images return a list with a single item based on the values in docker_image_base. Wouldn't it be simpler and clearer to just store a flat list of images for each case (origin / ocp)?

Should this list be stored in the `docker_image_base` dict? Also, should it only apply to the values returned by `containerized_docker_images` and `rpm_images`, or should `qualified_docker_images` also follow this?